orangecontrib/educational/widgets/owpolynomialregression.py

```diff
@@ -266,6 +267,7 @@ def apply(self):
             data_table = Table(
                 Domain([attributes], class_vars=[class_var]), self.data)
 
+            data_table = IsDefined()(data_table)
```

Comment:
How about this a few lines below:
```py
            for preprocessor in learner.active_preprocessors:
                preprocessed_data = preprocessor(preprocessed_data)
```

Reply:
Even better.

Reply:
How do you know? I don't know what the original intention was. `self.preprocessors` is currently set to `None` and is later unused. @PrimozGodec can shed some light?